File: plugin/android/java/jar/android/plugin/src/main/java/com/test/TestPlugin.java
//======================================================================
// Project Name    : java
//
// Copyright © 2018 U-CREATES. All rights reserved.
//
// This source code is the property of U-CREATES.
// If such findings are accepted at any time.
// We hope the tips and helpful in developing.
//======================================================================
package com.test;
import android.util.Log;
public class TestPlugin {
    public int execute() {
        android.util.Log.i("UnityAndroidPlugin", String.format("%s.%s", new Object() {} .getClass().getName(), new Object() {} .getClass().getEnclosingMethod().getName()));
        return 1 + 1;
    }
}

File: plugin/android/kotlin/jar/android/plugin/src/main/java/com/test/TestPlugin.kt
//======================================================================
// Project Name    : kotlin
//
// Copyright © 2018 U-CREATES. All rights reserved.
//
// This source code is the property of U-CREATES.
// If such findings are accepted at any time.
// We hope the tips and helpful in developing.
//======================================================================
package com.test
import android.util.Log
import java.lang.StackTraceElement
import java.lang.Thread
class TestPlugin {
    fun execute(): Int {
        var elements: Array<StackTraceElement?> = Thread.currentThread().stackTrace
        var element: StackTraceElement? = elements[2]
        Log.i("UnityAndroidPlugin", String.format("%s.%s", element?.className, element?.methodName))
        return 1 + 1
    }
}